loader/src/injector/module.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <cstring>
```

Comment:
Why?

Reply:
Language server suggestion, because in
```cpp
void clearApi() { memset(&api, 0, sizeof(api)); } 
```
`memset` is defined in that header